src/features/users/user-controller.ts

```diff
@@ -169,13 +169,22 @@ class UserController {
         // I tried bcc: users.map(user => user.email) but I got an error that an email address was required
         const emailPromises = [];
         for (let i = 0; i < users.length; i++) {
+            const user = users[i];
+            const poorMansTemplate = `
+            Hello ${user.firstName} ${user.lastName},
+
+            You have received a message through Rederly:
```

Comment:
I'm iffy on the `through Rederly` part. It feels misleading because we don't have an in-app messaging system. But I don't have a better suggestion right now.